make_wheels.py

```diff
@@ -134,20 +134,33 @@ def write_ziglang_wheel(out_dir, *, version, platform, archive):
         name='ziglang',
         version=version,
         tag=f'py3-none-{platform}',
-        metadata={
-            'Summary': 'Zig is a general-purpose programming language and toolchain for maintaining robust, optimal, and reusable software.',
-            'Description-Content-Type': 'text/markdown',
-            'License': 'MIT',
-            'Classifier': [
-                'License :: OSI Approved :: MIT License',
-            ],
-            'Project-URL': [
-                'Homepage, https://ziglang.org',
-                'Source Code, https://github.com/ziglang/zig-pypi',
-                'Bug Tracker, https://github.com/ziglang/zig-pypi/issues',
-            ],
-            'Requires-Python': '~=3.5',
-        },
+        metadata=[
+            ('Summary', 'Zig is a general-purpose programming language and toolchain for maintaining robust, optimal, and reusable software.'),
+            ('Description-Content-Type', "'text/markdown'; charset=UTF-8; variant=GFM"),
```

Comment:
The other variant is `CommonMark`: https://packaging.python.org/en/latest/specifications/core-metadata/#description-content-type, and I felt that `charset=UTF-8` is always good to add (not necessarily needed, though).